### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by RainRat**

### CODE REVIEW

File: lib/cfilters.c [hunk 1]
```diff
@@ -590,7 +590,7 @@ CURLcode Curl_conn_ev_data_idle(struct Curl_easy *data)
 
 /**
  * Notify connection filters that the transfer represented by `data`
- * is donw with sending data (e.g. has uploaded everything).
+ * is down with sending data (e.g. has uploaded everything).
```
Comment (bagder) [inline lib/cfilters.c:593]:
```suggestion
 * is done with sending data (e.g. has uploaded everything).
```

File: lib/cfilters.h [hunk 2]
```diff
@@ -451,7 +451,7 @@ CURLcode Curl_conn_ev_data_idle(struct Curl_easy *data);
 
 /**
  * Notify connection filters that the transfer represented by `data`
- * is donw with sending data (e.g. has uploaded everything).
+ * is down with sending data (e.g. has uploaded everything).
```
Comment (bagder) [inline lib/cfilters.h:454]:
```suggestion
 * is done with sending data (e.g. has uploaded everything).
```

**Review by bagder — COMMENTED** [review]
Apart from these two nits, it seems entirely correct!

File: lib/cfilters.h (hunk 2 above)
Comment (RainRat) [reply lib/cfilters.h]: ok, that also makes sense

**Comment by bagder** [comment]
@RainRat can you please squash this commit frenzy into a single commit and force-push?

**Pull request closed without merging**